Tensile/Source/lib/include/Tensile/ExactLogicLibrary.hpp

```diff
@@ -213,6 +213,33 @@ namespace Tensile
         }
     };
 
+    template <typename MyProblem, typename MySolution>
+    struct PerCUorOverallLibrary
```

Comment:
Yup, Scott is right. I originally made it distinct to future-proof for changes that would need to be made to support the API for selecting perCU or overall, but now I know this can/will be handled by a switch in ContractionProblem.